core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreMainActivity.java

```diff
@@ -1563,15 +1565,7 @@ private void loadPrefs() {
     }
 
     openFullScreenIfEnabled();
-    updateNightMode();
-  }
-
-  private void updateNightMode() {
```

Comment:
this method shouldn't have been deleted, it should have invoked the NightModeViewPainter

this deletion has resulted in duplicating this call `painter.update(getCurrentWebView(), videoView);`